packages/labs/eslint-plugin/tsconfig.json

```
@@ -4,10 +4,10 @@
     "composite": true,
     "tsBuildInfoFile": "tsconfig.tsbuildinfo",
     "target": "es2020",
```

Since this caught my eye, the rest of our packages all do es2021.
```suggestion
    "target": "es2021",
```